### PR TITLE
Updates for v1.1: AboutUs and Developer Guide

### LINKED ISSUE #19: DG - glossary

### LINKED ISSUE #20: AboutUs page

### PR DESCRIPTION
**Pull request by jiaqi20**

### CODE REVIEW

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Review by caitlinjee — APPROVED** [review]
lgtm

**Review by daongochieu2810 — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -438,7 +438,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
-* **Private contact detail**: A contact detail that is not meant to be shared with others
+* **Fridge**: A personalised storage that contains all the ingredients that a user has
+* **Recipe**: A set of cooking instructions that describes how to prepare a healthy meal and ingredients required
+* **Ingredient**: Foods that can be used to prepare a particular healthy dish according to a recipe
```
Comment (oliviajohansen) [inline docs/DeveloperGuide.md:443]: maybe don't put 'healthy'? it restricts the types of recipes unnecessarily

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -438,7 +438,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+* **Consumption**: A counter which calculates and displays the calorie intake of a user based on the recipes consumed
```
Comment (oliviajohansen) [inline docs/DeveloperGuide.md:444]: same as above

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -438,7 +438,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
-* **Private contact detail**: A contact detail that is not meant to be shared with others
+* **Fridge**: A personalised storage that contains all the ingredients that a user has
+* **Recipe**: A set of cooking instructions that describes how to prepare a healthy meal and ingredients required
```
Comment (oliviajohansen) [inline docs/DeveloperGuide.md]: maybe don't put 'healthy'? it restricts the types of recipes unnecessarily

**Review by oliviajohansen — APPROVED** [review]
thanks lgtm

**Pull request merged**